File: learn/drill/programming_in_th/0003/0003.c
#include <stdio.h>

int main(void)
{
    int m = 0, n = 0, metrices = 2, round = metrices;
    
    scanf("%d %d", &m, &n);

    int buf = 0, res[m][n];

    for (int i = 0; round > 0; ++i)
    {
        if (i == m)
        {
            round--;
            i = 0;
            if (!round)
            {
                break;
            }
        }

        for (int j = 0; j < n; ++j)
        {
            scanf("%d", &buf);

            if (round == metrices)
            {
                res[i][j] = buf;
            }
            else
            {
                res[i][j] += buf;
            }
        }
    }

    for (int i = 0; i < m; ++i)
    {
        for (int j = 0; j < n; ++j)
        {
            printf("%d ", res[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: learn/drill/programming_in_th/0003/0003.py
metrices = 2
round = metrices

stdin = input().split(' ')

m = int(stdin[0])
n = int(stdin[1])

res = [[0 for i in range(0, m)] for j in range(0, n)]

i = 0
while round > 0:
    if i == m:
        round -= 1
        i = 0
        if not round:
            break
    
    stdin = input().split(' ')

    for j in range(0, n):
        res[i][j] += int(stdin[j])

    i += 1

for d in range(0, m):
    print(*res[d])